src/main/target/common_pre.h

```diff
@@ -287,5 +287,6 @@
 #define USE_CMS_FAILSAFE_MENU
 #define USE_SMART_FEEDFORWARD
 #define USE_TELEMETRY_SENSORS_DISABLED_DETAILS
+#define USE_VTX_TABLE
```

Comment:
This is a case where I'd go for making this universally available for all targets that support VTX - otherwise we'll have to support two different ways of controlling a VTX going forward, and we are still going to expose ourselves to being held accountable for frequency / licensing management for some targets.
But I'll merge for now so further work gets unblocked, we can look into making this fit onto all targets in a separate track.

Reply:
We can review the availability once all VTX drivers are converted; Tramp was easy, but anticipating SA major overhaul.